extension/src/popup/components/account/AccountAssets/index.tsx

```diff
@@ -285,7 +281,7 @@ export const AccountAssets = ({
             </div>
             <div className="AccountAssets__copy-right">
               <div className="asset-amount" data-testid="asset-amount">
-                {formatAmount(amountVal)} <span>{amountUnit}</span>
+                {formatAmount(amountVal)}
```

Comment:
This commit drops the ticker as suggested by @JakeUrban [here](https://github.com/stellar/freighter/pull/1737#issuecomment-2539999180):
![Screenshot 2025-01-14 at 16 20 36](https://github.com/user-attachments/assets/ea6c8982-c7fd-47f6-bde1-03cfde9fa894)

The "N/A" label has also been removed on this older commit [here](https://github.com/stellar/freighter/pull/1785/commits/fcfb900be026eeb112ee6e995515b5ff2917d014).